pt/lessons/specifics/ecto.md

```diff
@@ -0,0 +1,368 @@
+---
+layout: page
+title: Ecto
+category: specifics
+order: 2
+lang: pt
+---
+
+Ecto é um projecto oficial do Elixir que fornece um invólucro de base de dados e
```

Comment:
Thanks for the suggestion. As I am Angolan. Angola has not ratified the new agreement spelling.